js/plugin/popup-dialog.ts

```diff
@@ -210,51 +179,240 @@ export class BrowserModPopup extends LitElement {
     this._styleAttributes = structuredClone(this._styleAttributes);
   }
 
-  _setStyleAttribute(newStyle) {
+  _setStyleAttribute = (newStyle: string): void => {
     this._updateStyleAttributes(newStyle);
-    this._styleSequenceIndex = this._styleSequence.indexOf(newStyle);
+    const index = this._styleSequence.indexOf(newStyle);
+    this._styleSequenceIndex = index >= 0 ? index : 0;
   }
 
-  _cycleStyleAttributes(dir: string = "forward") {
+  _cycleStyleAttributes = (dir: string = "forward"): void => {
+    const length = this._styleSequence.length;
+    if (length === 0) return;
+
     if (dir === "forward") {
-      this._styleSequenceIndex = (this._styleSequenceIndex + 1) % this._styleSequence.length;
+      this._styleSequenceIndex = (this._styleSequenceIndex + 1) % length;
```

Comment:
Sorry. This makes the code less readable and is irrelevant to your PR.

Reply:
It is a matter of personal preference, but okay